server/me.go

```diff
@@ -36,7 +36,7 @@ func newMeResponse(usr *chronograf.User) meResponse {
 	}
 }
 
-func getEmail(ctx context.Context) (string, error) {
+func getUsername(ctx context.Context) (string, error) {
```

Comment:
would it be useful to combine `getUsername`, `getProvider`, and the possibly forthcoming `getScheme` into a single function that returns pointers to any gotten values? or an over-optimization at this point?

Reply:
Good question. I don't have strong feelings either way. I think I'd leave it this way for now, but thats just me.